sdk/src/wallet/migration/migrate_0.rs

```diff
@@ -88,11 +88,11 @@ impl Migration for Migrate {
         if let Some(mut wallet) = storage.get::<serde_json::Value>(WALLET_INDEXATION_KEY).await? {
             ConvertHrp::check(
                 wallet
-                    .get_mut("clientOptions")
+                    .get_mut("client_options")
                     .ok_or(Error::Storage("missing client options".to_owned()))?
                     .get_mut("protocolParameters")
                     .ok_or(Error::Storage("missing protocol params".to_owned()))?
-                    .get_mut("bech32Hrp")
+                    .get_mut("bech32_hrp")
```

Comment:
Why are some fields camelCase and others snake_case?

Reply:
Because some of the structs have been renames and some haven't. I think we should change them, but every rename is a migration too.